NETCORE/test/FunctionalTests.Utils/FunctionalTests.Utils.csproj

```diff
@@ -3,10 +3,10 @@
   <PropertyGroup>
     <VersionPrefix>1.0.2</VersionPrefix>
     <TargetFrameworks>net461;netcoreapp2.0</TargetFrameworks>
```

Comment:
can this be changed: net461;netcoreapp2.1;netcoreapp3.1

Reply:
just updated. thanks for reviewing